docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+git stash
+git pull
```

Comment:
Would something along these lines work for unattended?
https://julienrenaux.fr/2013/10/04/how-to-automatically-checkout-the-latest-tag-of-a-git-repository/